rt_bi_emulator:
  av1:
    ros__parameters:
      robotId: 1
      updateInterval: 1.8
      timeSecs:
        - 0.0
        - 50.0
      cutOffTime: inf
      saPose:
        - "[120.804, 134.190, -13.91]"
        - "[120.148, 133.236, -93.91]"
      fov:
        - "[[283.602, 65.354], [296.297, 119.257], [120.804, 134.190]]"
        - " [[198.3992242389282, -212.9295365640743], [251.42809157912083, -196.97222906511783], [176.61394893383383, -37.52433287877531]]"
  av2:
    ros__parameters:
      robotId: 2
      updateInterval: 1.3
      timeSecs:
        - 0.0
        - 50.0
      saPose:
        - "[342.837, -48.842, 93.91]"
        - "[343.492, -47.888, -13.91]"
      fov:
        - "[[180.038, 19.995], [167.343, -33.908], [342.837, -48.842]]"
        - "[[72.70282205133017, 165.91498164654277], [113.03668939152276, 127.96928914549929], [213.71691277163958, 272.48455135724555]]"
